app/src/main/java/net/kwatts/powtools/MainActivity.java

```diff
@@ -390,6 +391,25 @@ private void showEula() {
                 .show();
     }
 
+    private void showDonation() {
+        new MaterialDialog.Builder(this)
+                .title("Donate")
+                .content("This app is openly developed and maintained by contributors in their spare time. Show support and help fuel their OneWheel addictions :)")
+                .items(R.array.donation_options)
+                .itemsCallback(new MaterialDialog.ListCallback() {
+                    @Override
+                    public void onSelection(MaterialDialog dialog, View view, int which, CharSequence text) {
+                        if (which == 0) {
+                            Intent browserIntent = new Intent(Intent.ACTION_VIEW, Uri.parse("https://www.paypal.com/cgi-bin/webscr?cmd=_donations&business=kwatkins%40gmail%2ecom&lc=US&item_name=pOneWheel%20Android%20App&no_note=1&no_shipping=1&currency_code=USD&bn=PP%2dDonationsBF%3abtn_donateCC_LG%2egif%3aNonHosted"));
+                            startActivity(browserIntent);
+                        } else if (which == 1) {
+                            //TODO: Google Play
+                        }
+
+                    }
+                })
+                .show();
```

Comment:
this dialog doesn't work in night mode, just looks like a white box. I think the text is getting set to white but the background isn't getting set to dark and it should be.

Reply:
Good catch.